crates/lumina_client/src/ui/game_ui/health.rs

```diff
@@ -1,7 +1,14 @@
 use bevy::prelude::*;
+use lumina_common::prelude::*;
+use lumina_shared::prelude::*;
+use lumina_shared::{
+    health::{Health, MaxHealth},
+    player::PlayerId,
+};
+
 use velyst::prelude::*;
 
-use crate::ui::game_ui::GameUi;
+use crate::{player::LocalPlayerId, ui::game_ui::GameUi};
```

Comment:
```suggestion
use crate::player::LocalPlayerId;
use crate::ui::game_ui::GameUi;
```

I like it split if the modules are different haha. (minor changes)